<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QDoubleSpinBox" name="APM">
     <property name="prefix">
      <string>Area per molecule (A^3): </string>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.200000000000000</double>
     </property>
     <property name="value">
      <double>60.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDoubleSpinBox" name="head_solvent">
     <property name="prefix">
      <string>head solvent SLD: </string>
     </property>
     <property name="minimum">
      <double>-2.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>6.360000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDoubleSpinBox" name="tail_solvent">
     <property name="prefix">
      <string>tail solvent SLD: </string>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>6.360000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDoubleSpinBox" name="thick_h">
     <property name="prefix">
      <string>thickness heads (A): </string>
     </property>
     <property name="maximum">
      <double>300.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>9.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDoubleSpinBox" name="V_h">
     <property name="prefix">
      <string>volume head group (A^3): </string>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>319.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLineEdit" name="b_h_real">
     <property name="text">
      <string>6.01e-4</string>
     </property>
     <property name="placeholderText">
      <string>head - sum real scattering lengths</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLineEdit" name="b_h_imag">
     <property name="text">
      <string>0</string>
     </property>
     <property name="placeholderText">
      <string>head - sum imag scattering lengths</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QDoubleSpinBox" name="thick_t">
     <property name="prefix">
      <string>thickness tails (A): </string>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>14.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QDoubleSpinBox" name="V_t">
     <property name="prefix">
      <string>volume tail group (A^3): </string>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>782.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLineEdit" name="b_t_real">
     <property name="text">
      <string>-2.92e-4</string>
     </property>
     <property name="placeholderText">
      <string>tail - sum real scattering lengths</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QLineEdit" name="b_t_imag">
     <property name="text">
      <string>0</string>
     </property>
     <property name="placeholderText">
      <string>tail - sum imag scattering lengths</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>tail - sum real scatt lengths (A)</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>tail - sum imag scatt lengths (A)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>head - sum real scatt lengths (A)</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>head - sum imag scatt lengths (A)</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="lipid_selector">
     <item>
      <property name="text">
       <string>item1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item2</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
